cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -86,11 +74,26 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
     ).convert { it.expandTilde() }
         .file(mustExist = false, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = false)
         .convert { it.absoluteFile.normalize() }
+        .outputGroup()
 
     private val outputFormats by option(
         "--output-formats", "-f",
         help = "The list of output formats to be used for the ORT result file(s)."
-    ).enum<FileFormat>().split(",").default(listOf(FileFormat.YAML))
+    ).enum<FileFormat>().split(",").default(listOf(FileFormat.YAML)).outputGroup()
+
+    private val rules by mutuallyExclusiveOptions(
+        option(
+            "--rules-file", "-r",
+            help = "The name of a script file containing rules."
+        ).convert { it.expandTilde() }
+            .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+            .convert { FileType(it.absoluteFile.normalize()) },
+        option(
+            "--rules-resource",
+            help = "The name of a script resource on the classpath that contains rules."
+        ).convert { StringType(it) },
+        name = "Rule Options"
```

Comment:
What do you think about extracting the name of the group to constants, like here "Rule Options", to eliminate duplication and for making it clear these must match?

Reply:
I thought about it but wasn't sure if I like it. But as you also propose it I will include this change in the follow-up PR if that's fine with you.

Reply:
Sure, was actually just an _optional_ proposal.